qiskit_ibm_runtime/runtime_program.py

```diff
@@ -115,6 +115,23 @@ def _format_common(schema: Dict) -> None:
                         + str(property_name in schema.get("required", []))
                     )
 
+        def _format_backend_requirements(schema: Dict) -> None:
+            """Add backend requirements details to `formatted`."""
+            if "input_allowed" in schema:
+                formatted.append(
+                    " " * 4 + "Input allowed: {}".format(str(schema["input_allowed"]))
+                )
+            if "min_num_qubits" in schema:
+                formatted.append(
+                    " " * 4
+                    + "Minimum number qubits: {}".format(str(schema["min_num_qubits"]))
+                )
+            if "supported_features" in schema:
+                formatted.append(
+                    " " * 4
+                    + "Supported features: {}".format(str(schema["supported_features"]))
+                )
```

Comment:
For any key that we have not handled here, we should call the sentence_case function. You might have to loop through the schema dict in this function.